management/utils.py

```diff
@@ -179,6 +179,9 @@ def wait_for_service(port, public, env, timeout):
 				return False
 		time.sleep(min(timeout/4, 1))
 
+def get_php_version():
+	return "8.1"
```

Comment:
Could you leave behind a new comment for the next developer that incrementing the version here alone isn't safe? (Like adding instructions how to increment PHP versions the safe way?)

Reply:
It's [here](https://github.com/mail-in-a-box/mailinabox/pull/2309/files#diff-f7b5f2e6c540e91e9b00a6933e0bfa708b863afa867140985dd1f174bac7894aR184)